rbdoom3bfg.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/bash
+
+# Set game folder paths
+rbdoom3bfg_data_dir=${XDG_DATA_HOME}/rbdoom3bfg
+rbdoom3bfg_base_dir=${rbdoom3bfg_data_dir}/base
+
+# Check data folder
+if [ ! -d ${rbdoom3bfg_data_dir} ]
+then
+    mkdir -p ${rbdoom3bfg_data_dir}
+fi
+
+# Check base folder
+if [ ! -d ${rbdoom3bfg_base_dir} ]
+then
+    zenity --width=400 --error --title="Game data files not found" \
```

Comment:
where are data files coming from? proprietary?

Reply:
Yes it is a commercial game from id Software. Users need to copy the game data of Doom 3 BFG either from Steam or GOG into ~/.var/app/io.github.RobertBeckebans.RBDoom3BFG/data/rbdoom3bfg/base/